src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    return one.div(two).toString();
+  }
+  if (operation === '%') {
+    return one.mod(two).toString();
+  }
+  throw Error(`Unknown operation '${operation}'`);
+}
```

Comment:
[OPTIONAL]

- When I tried to do something like `8 +` and then I pressed any operation again like `-`, it's better if it's changed into `8 -` removing the previous operation. But in your case, it's saying invalid which shows in the below screenshot. It's better if you work on that after that your app will be fully bug-free.

![Screenshot from 2021-09-15 20-50-11](https://user-images.githubusercontent.com/22434660/133462364-6015e05a-c185-462b-9ef4-12952d7ae9f9.png)